### PR TITLE
Make template caching configurable

### LINKED ISSUE #18: Add possibility to configure template caching

### PR DESCRIPTION
**Pull request by achimgrimm**

### CODE REVIEW

File: src/test/java/de/neuland/kafkabridge/lib/templating/TemplateRendererTest.java [hunk 1]
```diff
@@ -35,36 +48,111 @@ static void deleteTemplatePathFile() {
...
+        var result = templateRenderer.render(templatePath,
+                                             new Json<>("{}"));
+
+        assertThatJson(result.value()).isEqualTo("""
+                                                                 {
+                                                                   "data": "before"
+                                                                 }""");
+    }
+
+    private TemplateRenderer givenTemplateRenderer(Option<Duration> maybeCacheDuration) {
```
Comment (michel-zimmer) [inline src/test/java/de/neuland/kafkabridge/lib/templating/TemplateRendererTest.java:138]: Consider adding `givenTemplateRenderer()` and `givenTemplateRendererWithCaching(Duration)` overloads

File: src/test/java/de/neuland/kafkabridge/lib/templating/TemplateRendererTest.java [hunk 2]
```diff
@@ -35,36 +48,111 @@ static void deleteTemplatePathFile() {
...
+                          )]
+                        }""");
 
         // when
         var result = templateRenderer.render(templatePath,
                                              new Json<>("""
-                                                                {
-                                                                  "id": "1234"
-                                                                }"""));
+                                                                        {
```
Comment (michel-zimmer) [inline src/test/java/de/neuland/kafkabridge/lib/templating/TemplateRendererTest.java]: Consider adding our teams `.editorconfig` file

**Review by michel-zimmer — CHANGES_REQUESTED** [review]
Please change the style of the commit message to the one proposed at https://chris.beams.io/posts/git-commit/.

Consider changing it to

```
Make template caching configurable

Closes #18 
```

for example.

**Comment by michel-zimmer** [comment]
Also feel free to increment the app version in `kafka-bridge/charts/kafka-bridge/Chart.yaml`

**Comment by achimgrimm** [comment]
> Also feel free to increment the app version in `kafka-bridge/charts/kafka-bridge/Chart.yaml`

done

**Review by michel-zimmer — APPROVED** [review]

**Pull request merged**